sdk/communication/Azure.Communication.Administration/tests/PhoneNumberAdministrationClient/PhoneNumberAdministrationClientLiveTests.cs

```diff
@@ -29,6 +29,7 @@ public class PhoneNumberAdministrationClientLiveTests : PhoneNumberAdministratio
         /// <param name="isAsync">A flag used by the Azure Core Test Framework to differentiate between tests for asynchronous and synchronous methods.</param>
         public PhoneNumberAdministrationClientLiveTests(bool isAsync) : base(isAsync)
         {
+            Matcher.LegacyExcludedHeaders.Add("traceparent");
```

Comment:
Do these have to be excluded? I thought we always have trace headers.

Reply:
Yes this was needed for a few tests. Example:

Message: 
    Azure.Core.TestFramework.TestRecordingMismatchException : Unable to find a record for the request GET https://sanitized.communication.azure.com/administration/phonenumbers/phonenumbers?api-version=2020-07-20-preview1
    Header differences:
        <traceparent> is absent in request, value <00-7766485877d3914c84451c02a78b8d48-405560a140485448-00>
    Body differences:

Reply:
Does the method we call have a diagnostic scope?

Reply:
Yes, it does - https://github.com/Azure/azure-sdk-for-net/blob/e9453ac9a3eb7b3b957aa06b3f23044eff5c878f/sdk/communication/Azure.Communication.Administration/src/PhoneNumberAdministrationClient.cs#L87

Reply:
In the wrong place. It has to be wrapping the inner paging call.

Reply:
Welp, that would explain it.

Reply:
Filed: https://github.com/Azure/azure-sdk-for-net/issues/17633